NineChronicles.DataProvider/RenderSubscriber.cs

```diff
@@ -355,6 +436,69 @@ protected override Task ExecuteAsync(CancellationToken stoppingToken)
                                     SubRecipeId = combinationEquipment.subRecipeId ?? 0,
                                     BlockIndex = ev.BlockIndex,
                                 });
+                                if (combinationEquipment.payByCrystal)
+                                {
+                                    Currency crystalCurrency = new Currency("CRYSTAL", 18, minters: null);
```

Comment:
```suggestion
                                    Currency crystalCurrency = CrystalCalculator.CRYSTAL
```

Reply:
👍 I'll change other codes that uses the crystal currency.